cadctl/cmd/investigate/investigate.go

```diff
@@ -255,45 +254,30 @@ func isAWS(cluster *v1.Cluster) (bool, error) {
 // - the AWS account access flow is supported by CAD
 // Performs according pagerduty actions and returns whether CAD needs to investigate the cluster
 func clusterRequiresInvestigation(cluster *v1.Cluster, ocmClient *ocm.SdkClient, pdClient *pagerduty.SdkClient) (bool, error) {
+	// 1. Cluster is not in uninstalling state
 	if cluster.State() == v1.ClusterStateUninstalling {
```

Comment:
This might not even happen anymore. This usually happened for CHGMs resolving due to cluster uninstalls only. Doesn't hurt keeping it here though, we might hit this in rare cases with other alerts as well.